### ISSUE TITLE
StatementExecutorSpec failing

### ISSUE THREAD

**Issue by SethTisue** [scalikejdbc/scalikejdbc#841]
when I run the Scala 2.12 community build on my MacOS laptop I keep seeing:

```
[scalikejdbc] [info] StatementExecutorSpec:
[scalikejdbc] [info] StatementExecutor
[scalikejdbc] [info] - should work with JSR-310 APIs *** FAILED ***
[scalikejdbc] [info]   09:56:49.331 did not equal 10:56:49.331 (StatementExecutorSpec.scala:35)
```

looks like a timezone thing? here in San Francisco we just changed from standard time to daylight saving time a few days ago, maybe that's related?

**Comment by seratch** [comment]
Hmm. I agree that DST seems to be suspicious. We will check.

**Comment by seratch** [comment]
I found that the changes in PR https://github.com/scalikejdbc/scalikejdbc/pull/790 seem to be related.

**Comment by seratch** [comment]
@SethTisue This issue should be fixed by #847 thanks to @tanishiking . Could you check it out when you have time?

**Comment by SethTisue** [comment]
haven't seen this again 👍
